basics/realloc/native/tests/realloc.test.ts

```diff
@@ -1,126 +1,126 @@
-            console.log(`   Street:     ${enhancedAddressInfo.street}`);
-            console.log(`   City:       ${enhancedAddressInfo.city}`);
-            console.log(`   State:      ${enhancedAddressInfo.state}`);
-            console.log(`   Zip:        ${enhancedAddressInfo.zip}`);
-        };
+    await printEnhancedAddressInfo(testAccount.publicKey);
+    await printWorkInfo(testAccount.publicKey);
+  });
+
+  async function printAddressInfo(pubkey: PublicKey): Promise<void> {
+    await delay(2);
+    const data = (await client.getAccount(pubkey))?.data;
+    if (data) {
+      const addressInfo = AddressInfo.fromBuffer(Buffer.from(data));
+      console.log('Address info:');
+      console.log(`   Name:       ${addressInfo.name}`);
+      console.log(`   House Num:  ${addressInfo.house_number}`);
+      console.log(`   Street:     ${addressInfo.street}`);
+      console.log(`   City:       ${addressInfo.city}`);
     }
+  }
 
-    async function printWorkInfo(pubkey: PublicKey): Promise<void> {
-        await delay(2);
-        const data = (await connection.getAccountInfo(pubkey))?.data;
-        if (data) {
-            const workInfo = WorkInfo.fromBuffer(data);
-            console.log("Work info:");
-            console.log(`   Name:       ${workInfo.name}`);
-            console.log(`   Position:   ${workInfo.position}`);
-            console.log(`   Company:    ${workInfo.company}`);
-            console.log(`   Years:      ${workInfo.years_employed}`);
-        };
+  async function printEnhancedAddressInfo(pubkey: PublicKey): Promise<void> {
+    await delay(2);
+    const data = (await client.getAccount(pubkey))?.data;
+    if (data) {
+      const enhancedAddressInfo = EnhancedAddressInfo.fromBuffer(Buffer.from(data));
+      console.log('Enhanced Address info:');
+      console.log(`   Name:       ${enhancedAddressInfo.name}`);
+      console.log(`   House Num:  ${enhancedAddressInfo.house_number}`);
+      console.log(`   Street:     ${enhancedAddressInfo.street}`);
+      console.log(`   City:       ${enhancedAddressInfo.city}`);
+      console.log(`   State:      ${enhancedAddressInfo.state}`);
+      console.log(`   Zip:        ${enhancedAddressInfo.zip}`);
     }
+  }
 
-    function delay(s: number) {
-        return new Promise( resolve => setTimeout(resolve, s * 1000) );
+  async function printWorkInfo(pubkey: PublicKey): Promise<void> {
+    await delay(2);
+    const data = (await client.getAccount(pubkey))?.data;
+    if (data) {
+      const workInfo = WorkInfo.fromBuffer(Buffer.from(data));
+      console.log('Work info:');
+      console.log(`   Name:       ${workInfo.name}`);
+      console.log(`   Position:   ${workInfo.position}`);
+      console.log(`   Company:    ${workInfo.company}`);
+      console.log(`   Years:      ${workInfo.years_employed}`);
     }
-  });
-  
\ No newline at end of file
+  }
+
+  function delay(s: number) {
```

Comment:
Nice - love using a function for this rather than copy pasting everywhere. Just a tip I normally just call this `sleep` and give it a value in ms, then have a const SECONDS = 1000.

That way 
```
await sleep(2 * SECONDS);
```

Which makes the time super obvious.